handlers/formcontrols/multiSelect.cfc

```diff
@@ -30,6 +30,32 @@ component {
 			args.labels = ValueArray( args.records.label );
 		}
 
+
+		var sourceObject  = args.sourceObject ?: "";
+		var sourceIdField = presideObjectService.getIdField( sourceObject );
+
+		args.object   = args.relatedTo ?: "";
+		args.multiple = true;
```

Comment:
Line 38: this should not be hardcoded to true, it has been used for single select
args.multiple           = args.multiple         ?: true;

Line 42: Can you also include args.multiple check in the following if condition?